CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
 
-## [x.y.z] - a-b-c
-### Changed
-- Refactored tests for PHPUnit 10.x compat
```

Comment:
Shouldn't remove this one; it was not a stub, it was actual work done - just didn't get released yet because it was "only tests" and didn't affect consumers.
It should still be in the release notes.

Reply:
Of course. It's obvious now to me as well